hack/build/build-functest.sh

```diff
@@ -27,7 +27,7 @@ else
 fi
 (cd $test_path; go install github.com/onsi/ginkgo/ginkgo@latest)
 (cd $test_path; GOFLAGS= go get github.com/onsi/gomega)
-(cd $test_path; go mod tidy; go mod vendor)
+#(cd $test_path; go mod tidy; go mod vendor)
```

Comment:
you can just remove it, no use for having it commented out

Reply:
Done!